### PR TITLE
Minor fix based on input from CRAN

### PR DESCRIPTION
**Pull request by cosi1**

### CODE REVIEW

File: DESCRIPTION [hunk 1]
```diff
@@ -38,6 +38,6 @@ Suggests:
     curl,
     rmarkdown
 RoxygenNote: 6.1.0
-URL: https://repidemicsconsortium.org/epiflows
+URL: http://repidemicsconsortium.org/epiflows
```
Comment (zkamvar) [inline DESCRIPTION]: I think the url may need to be `http://www.repidemicsconsortium.org/epiflows`

**Comment by zkamvar** [comment]
Can you copy/paste the text they sent you?

File: DESCRIPTION (hunk 1 above)
Comment (cosi1) [reply DESCRIPTION]: Ah, yes, you're right. If the domain is preceded by `www.`, it works even with https://. (http://repidemicsconsortium.org/epiflows works as well, but redirects to https://www...)

Comment (zkamvar) [reply DESCRIPTION]:
```
$ ping repidemicsconsortium.org
PING repidemicsconsortium.org (85.233.160.22) 56(84) bytes of data.
^C
--- repidemicsconsortium.org ping statistics ---
6 packets transmitted, 0 received, 100% packet loss, time 5104ms
```

```
$ ping www.repidemicsconsortium.org
PING sni.github.map.fastly.net (185.199.110.153) 56(84) bytes of data.
64 bytes from 185.199.110.153: icmp_seq=1 ttl=54 time=80.7 ms
64 bytes from 185.199.110.153: icmp_seq=2 ttl=54 time=41.9 ms
64 bytes from 185.199.110.153: icmp_seq=3 ttl=54 time=52.4 ms
^C
--- sni.github.map.fastly.net ping statistics ---
3 packets transmitted, 3 received, 0% packet loss, time 2002ms
rtt min/avg/max/mdev = 41.963/58.386/80.764/16.392 ms

```

Comment (zkamvar) [reply DESCRIPTION]: I've also just turned on "Use https everywhere on the repo, but firefox still says that there are elements that aren't using https, so ¯\\\_(ツ)_/¯

**Review by zkamvar — APPROVED** [review]

File: DESCRIPTION (hunk 1 above)
Comment (cosi1) [reply DESCRIPTION]: Indeed :D 
https

**Comment by zkamvar** [comment]
You may merge when ready

**Comment by cosi1** [comment]
@zkamvar I've already merged :-)

**Pull request merged**